enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/JWTAuthenticator.java

```diff
@@ -411,6 +408,19 @@ private void checkTokenEnvAgainstDeploymentType(String keyType, APIConfig matche
         if (System.getenv("DEPLOYMENT_TYPE_ENFORCED") != null
                 && System.getenv("DEPLOYMENT_TYPE_ENFORCED").equalsIgnoreCase("false")
                 && keyType.equalsIgnoreCase(APIConstants.JwtTokenConstants.PRODUCTION_KEY_TYPE)) {
+            if (System.getenv("WHITELISTED_ORGANIZATIONS") != null) {
+                String[] whitelistedOrganizationsArray =
+                        System.getenv("WHITELISTED_ORGANIZATIONS").split("\\s+");
```

Comment:
What if we generate this array when the Enforcer is starting?

Reply:
Added a static block